docs/authentication.md

```diff
@@ -0,0 +1,45 @@
+# Authenticating to your Octopus Server
+
+There are two main ways of authenticating to your Octopus Server when using the Serverspec Extensions
+
+#### 1. Provide creds with each type
```

Comment:
Consider swapping 1 & 2 - I suspect 2 is more useful for most

Reply:
A good call. I like that idea.